test/reachsets.jl

```diff
@@ -103,3 +105,28 @@ end
     @test isequivalent(set(overapproximate(d, Zonotope)), set(R))
 
 end
+
+@testset "Taylor model reach-sets with non-float coefficients" begin
```

Comment:
maybe a matter of taste, but to me it's cleaner if we create the object we want to test directly:

```julia
Δt, orderT, orderQ = 0..1, 4, 3
set_variables(IA.Interval{Float64}, "x", order=orderQ, numvars=2)
p1 = Taylor1([0, (0..0.1) + (0..0.01)*x[2]], orderT)
p2 = Taylor1([0, (0..0.5) + (0..0.02)*x[1] + (0..0.03)*x[2]], orderT)
vec = [TaylorModel1(p1, zeroI, zeroI, Δt), TaylorModel1(p2, zeroI, zeroI, Δt)]
T = TaylorModelReachSet(vec, Δt)
H = set(overapproximate(T, Hyperrectangle))

@test isa(T, TaylorModelReachSet) && isa(H, Hyperrectangle)
```

Reply:
You are right.It looks more clear this way!